distribution/docs/src/main/resources/content/_installing/installing-from-the-command-line.adoc

```diff
@@ -20,12 +20,12 @@ After installing the Setup Type, these other components can be set up as describ
 ====== Configure Guest Claim Attributes
 
 The Guest Claim Attributes can be configured via the ${admin-console} after running the `profile:install` command.
-See <<_configuring_guest_claim_attributes,Configuring Guest Claim Attributes>>.
+See <<{managing-prefix}configuring_guest_claim_attributes,Configuring Guest Claim Attributes>>.
 
 ====== System Configuration Settings
 
-System Settings and Contact Info, as described in <<_configuring_system_settings, System Configuration Settings>>, can be changed in `<${branding}_HOME>/etc/system.properties`.
-The certificates must be set up manually as described in <<_managing_keystores_and_certificates,Managing Keystores and Certificates>>.
+System Settings and Contact Info, as described in <<{managing-prefix}configuring_system_settings, System Configuration Settings>>, can be changed in `<${branding}_HOME>/etc/system.properties`.
```

Comment:
Is this wrong? Change to `{home_directory}`.

Reply:
fixed